### DISCUSSION TITLE
Possibility of swapping the Horus X10S original internal RF module to a 4-in-1

### DISCUSSION THREAD

**Discussion by tzheng777** [EdgeTX/edgetx#5571] [General]
Hey all,

Originally I was wanting to run the multi wireless trainer on my Horus X10s (non-express) as master TX, however quickly realised the original internal XJT RF module can't run FRSKY-RX. As my aircraft now run ELRS, using an external 4-in-1 module is not an option.

Has anyone ever looked into replacing the internal XJT, which I believe looks like this ISRM board (https://m.media-amazon.com/images/I/51PXLwUwx0L.jpg) with any of the 4-in-1 boards? 

Just looking for ways to extend the life of the my trusty X10S, rather than having to finally buy a new controller!

**Comment by zshwei** [comment]
I also want to make my x10s to support elrs receiver without a external module (not convenient to carry ), hope there is a way.

**Comment by raphaelcoeffic** [comment]
AFAIK, the Radiomaster TX16 internal modules use the same pinout and should be available as spare parts. You might have to 3d print some adapter part as the mounting holes are different. It should fit inside the radio though. A few people have done the exact opposite: they fitted a X10 ISRM upgrade module inside a T16, which is very similar.
